projects/project-player-library/src/lib/pages/add-files-page/add-files-page.component.html

```diff
@@ -1,4 +1,10 @@
 <lib-header [title]="title"></lib-header>
+
+<div *ngIf="!taskId" class="flex rounded-2xl items-center flex-row gap-x-2 bg-rose-200 m-2 p-2">
```

Comment:
If the project doesn't have certificate this message shouldn't be shown
Where is that condition check?

Reply:
done